olGO/Views/Auth/LoginView.swift

```diff
@@ -81,7 +96,19 @@ class LoginView: UIView {
                         }
                     }, style: .modal)
                 }
-            })
-            .canceled(by: &bag)
+                
+                print("Login Response Status Code: \(response.statusCode)")
+                
+                if 200 ... 300 ~= response.statusCode {
+                    DispatchQueue.main.async {
+                        Navigate.shared.go(UIViewController {
+                                            UIView(backgroundColor: .white) {
+                                                Label("You are logged in")
+                                            }
+                        }, style: .modal)
+                    }
+                }
+        }
```

Comment:
Line 82: `.sink(receiveCompletion: { (result) in` needs to close `)`

Expected:
Line 111: `})`

Actual:
Line 111: `}`